modules/installation-creating-azure-service-principal.adoc

```diff
@@ -6,15 +6,11 @@
 [id="installation-creating-azure-service-principal_{context}"]
 = Creating a service principal
 
-The installation program requires an Azure identity to complete the installation. You can use a service principal.
-
-If you are unable to use a service principal, you can use a managed identity.
-
```

Comment:
what's the purpose to remove those information? I think it would be more clearer to leave it here.

Reply:
Removed this since it's repeated. The same has been mentioned in the Supported identities to access Azure resources section.

Reply:
Actually, the two parts are similar but not the same, so they might not duplicate. Just remind user that you can user either service principal or managed identity to create cluster.

Reply:
ok. Will restore this then.